test/calendar_test.js

```diff
@@ -0,0 +1,287 @@
+var expect = require('chai').expect;
+var Calendar = require('../src/data/calendar');
+
+const {utcDate, getFrenchBankHolidays, getBoxingDays} = require('../src/data/french-bank-holidays');
+global.utcDate = utcDate;
+global.getFrenchBankHolidays = getFrenchBankHolidays;
+global.getBoxingDays = getBoxingDays;
+
+
+describe("Calendar", () => {
+  describe("#monthWeekTable()", () => {
+    let weektable;
+
+    beforeEach(() => {
+      weekTable = Calendar.monthWeekTable(2017, 1);
+    });
+
+    it("should return a week array with the expected length", () => {
+      expect(weekTable)
+        .to.have.length.of(6);
+    });
+
+    it("should return a week array with the expected weeks length", () => {
+      expect(weekTable.every(week => week.length === 7))
+        .eql(true);
+    });
+
+    it("should return a week array with the null day type when not part of the month", () => {
+      weekTable.forEach(days => {
+        // ensure every day of each week has a null type whenever the day is not
+        // part of the target month
+        days.forEach(day => {
+          if (day.date.getMonth() !== 0) {
+            expect(day.type).to.be.a("null");
+          } else if (day.date.getDay() === 0) {
```

Comment:
do we want getUTCDay and getUTCMonth in tests ? I'm afraid this might fail in Travis sometimes otherwise